### ISSUE TITLE
Meziantou.Analyzer.Rules.UseStringCreateInsteadOfFormattableStringAnalyzer cannot be created

### ISSUE THREAD

**Issue by pmeems** [meziantou/Meziantou.Analyzer#363]
**Context**
Rule identifier: _MA0000_
Target Framework: _.NET 6.0_
Language version: _9.0_

**Describe the bug**
After updating from 1.0.708 to 1.0.717 I get this warning:
Warning	CS8032	An instance of analyzer Meziantou.Analyzer.Rules.UseStringCreateInsteadOfFormattableStringAnalyzer cannot be created from C:\Users\***\.nuget\packages\meziantou.analyzer\1.0.717\analyzers\dotnet\cs\Meziantou.Analyzer.dll: Could not load type 'Meziantou.Analyzer.Rules.UseStringCreateInsteadOfFormattableStringAnalyzer' from assembly 'Meziantou.Analyzer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'..

**To Reproduce**
Code or steps to reproduce the behavior:
Update to the latest version of the package

````c#
Code to reproduce the issue
````

**Additional context**
I'm using this package is a very simple Azure Functions v4 solution

I reverted back to version 1.0.708 and the warning disappears.
Next I updated 1 version at a time, so first to 1.0.709 and so on and the warning appears since v1.0.711

**Comment by meziantou** [comment]
This is a common issue, but not related to this analyzer. You can check previous issues for help:
- https://github.com/meziantou/Meziantou.Analyzer/issues/339
- https://github.com/meziantou/Meziantou.Analyzer/issues/275
- https://github.com/meziantou/Meziantou.Analyzer/issues/218
- https://github.com/meziantou/Meziantou.Analyzer/issues/152

**Comment by pmeems** [comment]
I just upgraded to v1.0.720 and got a different but similar warning.
After reading the suggested previous issues I also did a `dotnet build` outside VS2022 and now I don't have any warning anymore in VS2022 as well.

So this is an issue with VS. I'll close this issue here.
